egov-dss-dashboards/dashboard-analytics/MasterDashboardConfig.json

```diff
@@ -3339,15 +3339,15 @@
           "vizArray": [
             {
               "id": 2300,
-              "name": "DSS_HEALTH_COMPLAINTS_BY_DISTRICT",
-              "label": "DSS_HEALTH_COMPLAINTS_BY_DISTRICT",
+              "name": "DSS_HEALTH_COMPLAINTS_BY_HEALTH_DISTRICT",
```

Comment:
revert these master config changes @Taniya-eGov . we will write separate dashboard config for burundi dashboard.